website/src/content/deployment/example-deployment-now-v2.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: Example Deployment with Now v2
```

Comment:
Actually I'll be working on these soon, I just didn't have time before the release.

We can hide the link to the page but I also think that having the page "visible" helps users to know that there will be documentation about that topic.

Reply:
If you know you will add something there within say the next quarter, I think it's fine. Seeing how 'static server' page has said Coming Soon for close to a year now though, I would suggest that the default would be not to add it just because we know we want to do it at some point in the future.